deposit/bytecode-verification/lemmas.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+cat <<EOF
+requires "edsl.k"
+
+module LEMMAS
+    imports K-REFLECTION
+    imports EDSL
+EOF
+
+cat lemmas/macro.k \
+    lemmas/range.k \
+    lemmas/arith.k \
+    lemmas/buf.k \
+    lemmas/map.k \
+    lemmas/hash.k \
+    lemmas/gas.k
```

Comment:
Why not just importing them all from verification.k or a similar module and avoid extra scripts?

Reply:
dropped